roles/puppet_repositories/tasks/puppetlabs-6-debian.yml

```diff
@@ -0,0 +1,15 @@
+---
```

Comment:
there are `puppetX-relase-${DIST}.deb` files on https://apt.puppet.com/ and I wonder if using them in a similar way as we do with RPM would be a good idea.

Saves us the `apt_key` call as the deb already has the key in the right format, and also the installation onf `gpg`.